components/selection/selection-mixin.js

```diff
@@ -28,14 +35,26 @@ export class SelectionInfo {
 
 export const SelectionMixin = superclass => class extends superclass {
 
+	static get properties() {
+		return {
+			/**
+			 * Whether the selection control limited to single selection.
+			 */
+			singleSelect: { type: Boolean, attribute: 'single-select' }
```

Comment:
Since the mixin is called the "SelectionMixin" I might personally lean towards something like `selection-single` just to namespace all its public attributes (of which this is the only one for now) with its name. But I'm good either way.

Reply:
Yep... good point.  Will change.